lib/cdo/shared_constants/pd/shared_workshop_constants.rb

```diff
@@ -19,20 +19,25 @@ module SharedWorkshopConstants
     ].freeze
 
     SUBJECT_TEACHER_CON = 'Code.org TeacherCon'.freeze
+
+    # Academic Year Workshop subjects shared between CSP and CSD.
+    SUBJECT_WORKSHOP_1 = 'Academic Year Workshop 1'.freeze
+    SUBJECT_WORKSHOP_2 = 'Academic Year Workshop 2'.freeze
+    SUBJECT_WORKSHOP_3 = 'Academic Year Workshop 3'.freeze
+    SUBJECT_WORKSHOP_4 = 'Academic Year Workshop 4'.freeze
+    SUBJECT_WORKSHOP_5 = 'Academic Year Workshop 1 + 2'.freeze
```

Comment:
maybe rename these last two to `SUBJECT_WORKSHOP_1_2` and `SUBJECT_WORKSHOP_3_4`? 5 and 6 are a little confusing.

Reply:
Ah I see we used the 1-6 version last year. I think we can be more explicit this year since the names are simpler

Reply:
Yeah agree these are confusing, I wasn't 100% confident in my find and replace for these as they get moved into JS constants as well, so I kind of kicked the can. I can update though.

Reply:
Do you think maybe there's a benefit to leaving as-is, such that they're tied to the `LEGACY_SUBJECT_CSP_WORKSHOP_1`, etc? Or could change the names of the legacy variables as well...

Reply:
Meh actually looks like we've updated these variable names a few times, so not sure there's much historical continuity to maintain. I'll just try to do what makes sense for now.